arangod/Scheduler/Scheduler.cpp

```diff
@@ -418,12 +418,23 @@ std::string Scheduler::infoStatus() {
          " (<=" + std::to_string(_maxFifoSize[FIFO8]) + ")";
 }
 
-bool Scheduler::canPostDirectly() const noexcept {
+bool Scheduler::canPostDirectly(RequestPriority prio) const noexcept {
   auto counters = getCounters();
   auto nrWorking = numWorking(counters);
   auto nrQueued = numQueued(counters);
 
-  return nrWorking + nrQueued <= _maxQueueSize;
+  switch (prio) {
+    case RequestPriority::HIGH:
+//      return nrWorking + nrQueued <= _maxQueuedHighPrio;
```

Comment:
Is the commented code still needed here? If not, it should be removed.

Reply:
It has been subsequently removed in a different PR. Was left in this PR to point out change during anticipated merge of a different PR.